spa/src/pages/Connected/index.tsx

```diff
@@ -57,74 +58,106 @@ const Connected = () => {
 		AP.navigator.go( "addonmodule", { moduleKey: "gh-addon-admin" });
 	};
 
-	return (<Wrapper>
-		<ConnectedContainer>
-			<div>
-				<HeaderImg src={colorMode === "dark" ? "/public/assets/jira-github-connected-dark-theme.svg" : "/public/assets/jira-github-connected.svg"} alt=""/>
-				<Title>GitHub is connected!</Title>
-				<TopContent>
-					It’s time to let everyone know GitHub’s ready to use and your<br />
-					team can use issue keys to link work to Jira.<br />
-				</TopContent>
-				<ButtonContainer>
+	return (
+		<Wrapper>
+			<div css={connectedContainerStyle}>
+				<div>
+					<img
+						css={headerImgStyle}
+						src={
+							colorMode === "dark"
+								? "/public/assets/jira-github-connected-dark-theme.svg"
+								: "/public/assets/jira-github-connected.svg"
+						}
+						alt=""
+					/>
+					<h2 css={titleStyle}>GitHub is connected!</h2>
+					<div css={topContentStyle}>
+						It’s time to let everyone know GitHub’s ready to use and your
+						<br />
+						team can use issue keys to link work to Jira.
+						<br />
+					</div>
+					<div css={buttonContainerStyle}>
+						<Button
+							style={{ paddingLeft: 0 }}
+							appearance="link"
+							onClick={() => navigate("/spa/steps")}
+						>
+							Add another organization
+						</Button>
+					</div>
+				</div>
+				<div>
+					<Heading level="h500">What's next?</Heading>
+					<div css={flexWrapperStyle}>
+						<div css={sectionStyle}>
+							<img
+								css={sectionImgStyle}
+								src="/public/assets/github-integration.svg"
+								alt=""
+							/>
+							<Heading level="h400">Add issue keys in GitHub</Heading>
+							<div css={paragraphStyle}>
+								Include issue keys in pull request
+								<br />
+								titles, commit messages and
+								<br />
+								more to bring them into Jira.
+							</div>
+							<a
+								href="https://support.atlassian.com/jira-software-cloud/docs/reference-issues-in-your-development-work/"
+								target="_blank"
+							>
+								Learn about issue linking
+							</a>
+						</div>
+						<div css={sectionStyle}>
+							<img
+								css={sectionImgStyle}
+								src="/public/assets/collaborate-in-jira.svg"
+								alt=""
+							/>
+							<Heading level="h400">Collaborate in Jira</Heading>
+							<div css={paragraphStyle}>
+								Your team's development work
+								<br />
+								will appear in issues and the
+								<br />
+								code feature.
+							</div>
+							<a
+								href="https://support.atlassian.com/jira-cloud-administration/docs/integrate-with-development-tools/"
+								target="_blank"
+							>
+								Learn about development work in Jira
+							</a>
+						</div>
+					</div>
+				</div>
+				<div css={paragraphStyle}>
+					We're backfilling your organization's repositories into Jira (this{" "}
```

Comment:
Any reason why you've used `{" "}` instead of just a space? It's also right before a `<br />` so I'm not sure if it's needed

Reply:
@kamaksheeAtl  can you please remove this space, I dont think we need this space here.

Reply:
Okay I will